.drone.star

```diff
@@ -114,9 +114,12 @@ def main(ctx):
                     check_starlark(build_trigger) + \
                     changelog(ctx, trigger = build_trigger) + \
                     unit_test_pipeline(ctx, "clang", "clang++", "Debug", "Ninja", trigger = build_trigger) + \
-                    gui_test_pipeline(ctx, trigger = build_trigger) + \
                     gui_test_pipeline(ctx, trigger = build_trigger, server_version = ocis_server_version, server_type = "ocis")
 
+        # run gui tests against oc10 server only if the build title contains "gui-tests"
+        if "gui-tests" in ctx.build.title.lower():
```

Comment:
Would suggest to make it explicit with `oc10-gui-tests`.

Reply:
true :+1: